.jules/sentinel.md

```
@@ -0,0 +1,4 @@
+## 2024-05-14 - SSE Stream Endpoint Path Traversal
```

markdownlint-cli2 (0.21.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)